susanssky/careless-whisper
# 15-upload-transcription

This is a:






### Description



- **Purpose** - 
Add ability to upload transcripts to the database



- **How to check** - 
1. `npx prisma migrate dev --name init` or `npx prisma db push`
2. send the request that POST `localhost:3000/api/transcription/upload` by postman and put the below JSON content
```json
[{
    "lineNumber": "5",
    "startTime": "00:00:15,960",
    "endTime": "00:00:18,060",
    "content": "in a day we did"
},
{
    "lineNumber": "6",
    "startTime": "00:00:16,960",
    "endTime": "00:00:20,060",
    "content": "in a day we did"
}]
```
3. run `npx prisma studio` to see the above JSON 

### Links
- Story: #15 


### Author checklist




- [ ] I have explicitly tested the access rights for _at least_ unauthenticated and admin cases
